.github/workflows/test.yml

```diff
@@ -34,6 +34,12 @@ jobs:
           check-latest: true
           node-version-file: ".nvmrc"
 
+      - name: Disable GitHub Actions Annotations
+        run: |
+          echo "::remove-matcher owner=tsc::"
+          echo "::remove-matcher owner=eslint-compact::"
+          echo "::remove-matcher owner=eslint-stylish::"
+
```

Comment:
annotations for tsc and aslant are set up by `actions/setup-node` https://github.com/actions/setup-node/blob/5b32c9063c6eb277dacbf3e916a8bc5d2617c5b7/src/main.ts#L68-L74